server/lib/explore/related.py

```diff
@@ -89,6 +89,9 @@ def compute_related_things(state: ftypes.PopulateState,
       # We found a topic, so break!
       break
 
+  if dc != "sdg":
```

Comment:
`if params.is_sdg(state.uttr.insight_ctx):` ?

Because it might be sdg or sdgmini

Reply:
Done